### ISSUE TITLE
Client crash on FTB Infinity when Dungeon Guardian is near

### ISSUE THREAD

**Issue by geminimiche** [MrComputerGhost/Runic-Dungeons#25]
re: java.lang.IllegalArgumentException: Default value cannot be lower than minimum value!

Using recommended version of Infinity (1.11.2)

http://pastebin.com/qbEU22gX

Once it happens, user must have position reset via nbtexplorer to be able to log back in. It happens in dimensions other than just the dungeon dim.

**Comment by MrComputerGhost** [comment]
And you're certain this is a Runic Dungeon crash if it happens outside the dungeon dimension?

**Comment by KasperFranz** [comment]
EntityGuardian['Dungeon Guardian'] is the thing there is on all the crash logs (counting 7+ atm).

**Comment by geminimiche** [comment]
I'm fairly certain; all the crash logs have that in common - the dungeon guardian appears in the entity list near the player location at the time of crash in every single crash report so far. note this is a client side crash on a server - all we see on the console is an i/o error/disconnect for those users.

**Comment by MrComputerGhost** [comment]
Alright

**Comment by dj-party23** [comment]
I am experiencing the same bug also on FTB Infinity 1.11.2 
Here I have a good crash log where my client crashed directly after loading the Dungeon Guardian. http://pastebin.com/9tGUwDA9 
Disabling the mod only server side won't fix it. Only after disabling it in the client I can join without crashing.

**Comment by KasperFranz** [comment]
when can we expect a fix for this?

**Comment by geminimiche** [comment]
Why is there no way to disable the mob spawning while this is worked out? Also, this:  http://forum.feed-the-beast.com/threads/enderio-net-minecraft-entity-ai-attributes-rangedattribute-init.125403/  might provide more illumination.

**Comment by MrComputerGhost** [comment]
There will be a fix out this weekend. Everything should be solved.

**Comment by MrComputerGhost** [comment]
The update will be a couple days late due to me being sick with a stomach bug.

**Comment by MrComputerGhost** [comment]
Should be fixed now as of 1.1.6a.

**Comment by KasperFranz** [comment]
Is it pushed to Curse so it's possible for FTB to use :)?
(as far as i know it's where they pull there stuff from) 

http://minecraft.curseforge.com/projects/draconic-evolution/files

**Comment by MrComputerGhost** [comment]
Yes, it's on Curse

**Comment by KasperFranz** [comment]
I dont see it on curse yet and Infinity is still using the snapshot 5 (which is the latest on curse)

And sorry for keep bringing this up

**Comment by Dehietyi** [comment]
@MrComputerGhost I think this is still a problem even in 1.1.6a. http://pastebin.com/ELrrYM5w As soon as I removed Runic it stopped crashing in the affected areas. When I put it back into the Server and Client pack it happened again.

**Comment by Dehietyi** [comment]
Also happens on 1.1.5c http://pastebin.com/0kzCLVAM 
Also happens on 1.1.4 http://pastebin.com/7x8qnZ4X
Personally I think this isn't your mod. I think this is an interaction between your mod another that has been updated recently. I didn't have these issues with 1.1.5c or 1.1.4 when I had them in the older client and server packs. Also I updated to Java 8 on the client and server. Would Java 8 or any of the Java arguments (-XX:+UseG1GC) possibly cause this? Aside from mod updates and Java 8 (because of some of the mods) those are the only things I changed.
